wandb/proto/wandb_internal.proto

```diff
@@ -493,6 +493,7 @@ message Request {
     RunStartRequest       run_start = 11;
     CheckVersionRequest   check_version = 12;
     LogArtifactRequest    log_artifact = 13;
+    DownloadArtifactRequest    download_artifact = 14;
```

Comment:
Nit: align the field names above & below. I don't like this style because it results in those unnecessary changes, but we gotta follow the existing style.

Reply:
Are you sure? I don't see this being followed by `GetSystemMetricsRequest` or `GetSystemMetricsResponse` , `InternalMessagesResponse`

Reply:
Are you sure? I don't see this being followed by `GetSystemMetricsRequest` or `GetSystemMetricsResponse`